reana-ui/src/pages/workflowList/components/WorkflowSearch.js

```diff
@@ -0,0 +1,28 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React from "react";
+
+import { Input } from "semantic-ui-react";
+import debounce from "lodash/debounce";
+
+const TYPING_DELAY = 1000;
+
+export default function WorkflowSearch({ search }) {
+  const handleChange = debounce(search, TYPING_DELAY);
+  return (
+    <Input
+      fluid
+      icon="search"
+      placeholder="Search..."
+      onChange={(event, data) => handleChange(data.value)}
```

Comment:
```suggestion
      onChange={(_, data) => handleChange(data.value)}
```

Reply:
I'm not sure that using underscore for unused parameters are the best idea. I mean it's nice convention to indicate it, but in the same time we loose the information about what data it is (in case we will need it in the future). Also, it might be confusing when there are multiple unused parameters. What do you think about naming it as `_event`?

Reply:
In these kind of events it's kind of a convention that `event` is always the first arg. In any case, don't worry too much, it's not very important, it's more about avoiding the eslint warning of "Unused variable" more than other thing 😅